include/ba/SparseBlockMatrixOps.h

```diff
@@ -406,11 +406,13 @@ static void LoadDenseFromSparse(const SparseMatrix& sparse,
   DenseMatrix& dense = const_cast< DenseMatrix& >(dense_mat);
 
   typedef typename SparseMatrix::Scalar BlockType;
-  /*const int block_rows = BlockType::RowsAtCompileTime;*/
-  /*const int block_cols = BlockType::ColsAtCompileTime;*/
+  #ifndef NDEBUG
+  const int block_rows = BlockType::RowsAtCompileTime;
```

Comment:
I think it was commended because they generate a warning in Release. However it wont even compile in debug with those lines commented out.

This is the only place I included a NDEBUG. Of all the debug preprocessor flags this seems to be the most reliable.